controllers/keystoneapi_controller.go

```diff
@@ -654,9 +657,24 @@ func (r *KeystoneAPIReconciler) reconcileNormal(ctx context.Context, instance *k
 	if instance.Status.ReadyCount > 0 {
 		instance.Status.Conditions.MarkTrue(condition.DeploymentReadyCondition, condition.DeploymentReadyMessage)
 	}
-
 	// create Deployment - end
 
+	// create CronJob
+	cronjobDef := keystone.CronJob(instance, serviceLabels, serviceAnnotations)
+	cronjob := cronjob.NewCronJob(
+		cronjobDef,
+		time.Duration(5)*time.Second,
+	)
+
+	ctrlResult, err = cronjob.CreateOrPatch(ctx, helper)
+	if err != nil {
+		// TODO: Do we need a different status ?
```

Comment:
I guess the question is whether we log this action anywhere else? If we run it from within a job pod, it probably won't be writing to the normal service log files write? So if a user (or someone supporting the environment) needed to check the last time `trust_flush` was executed, if they can't get that information from a log file. Then it would probably be useful to report something in the status field when the job succeeds or fails?

Reply:
There are two levels of failures we have to consider
- Failure while creating a CronJob resource
- Failure in actual jobs triggered by CronJob
and operator would be responsible with only one and checking actual jobs would be out of its scope.

I have to double-check this but in my understanding the error is recorded in operator logs if the operator fails to create a cron job. The question is more about whether we should expose the failure as part of CR status.

Reply:
I think it's probably worthwhile setting a status.condition for it. Just for convenience sake, if it fails to create the cronjob it could go unnoticed by users if it's just hidden in the logs of the operator. imo, it would be worth adding a status.condition for it. But happy to wait for opinions from others as well.

Reply:
yes, we should have a  condition for the creating the CronJob resource it's part of setting up the service with its dependencies. 

If we can get the status, maybe just run time and successful/failed of the last run, also in a separate condition, this would make the condition the first spot to start with debugging. But then we should exclude this last run condition for calculate the overall Ready state of the service.

Reply:
re the overall Ready state, at least until it was running for the first time and as not an unknown state.

Reply:
I'd be careful with adding logic to check succesfull run because it might take long until the job is triggered for the first time(eg. if the job is executed once a day then it might take 24 hours in the worst case until the job is run.)

I think we can start with operators taking care of resource creation and let users to use cronjob/job interfaces directly to check how jobs are executed.

Reply:
makes sense

Reply:
Now the latest version should update the status according to result of cronjob resource creation.